packages/demo/examples/08-network/ExpandableNetwork.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint class-methods-use-this: 0, react/no-unused-prop-types: 0 */
+// import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Network,
+} from '@data-ui/network';
+
+import { getAddedGraph } from './data';
+
+
+class ExpandableNetwork extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = this.getStateFromProps(props);
+    this.onNodeClick = this.onNodeClick.bind(this);
```

Comment:
nit -- I'd vote for `onClickNode` name

Reply:
Since the rest of the event are on{noun}{verb}, maybe onNodeClick will be better?

Reply:
sounds good to me! 👍